test/libsolidity/util/TestFileParser.h

```diff
@@ -62,12 +62,30 @@ namespace test
 	/* special keywords */         \
 	K(Failure, "FAILURE", 0)       \
 
-enum class SoltToken : unsigned int {
-#define T(name, string, precedence) name,
-	SOLT_TOKEN_LIST(T, T)
-	NUM_TOKENS
-#undef T
-};
+namespace solt
```

Comment:
Why not soltest?

Reply:
Could be also soltest :) I don't have a strong opinion an that.

Reply:
Could we just *not* use any further namespace and still define ``Token``? We're in namespace ``dev::solidity::test`` already and the other ``Token`` is in ``dev::solidity``, isn't it? No, it's actually even in ``langutil`` now - in fact I'm not sure whether that that's the right place for the other one, but it means there's even less danger of conflict.

But to me this doesn't seem too important - I'm fine with ``solt``, ``soltest`` or no sub-namespace.